### PR TITLE
Episode 4

### PR DESCRIPTION
**Pull request by sockeqwe**
The issue for the discussion after releasing episode 4 must be created afterwards.

The link to the issue for the discussion after releasing episode 4 has been set to `/issues/36` which should be the correct number if no one else creates an issue in the mean time 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Before each episode we'll create an an [issue](https://github.com/artem-zinnatul
 
 ---
 
-Host: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin)
+Host: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin) & Hannes Dorfmann [sockeqwe](https://twitter.com/sockeqwe)
```
Comment (artem-zinnatullin) [inline README.md]: Hosts :)

File: show_notes/Episode_4.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+ - How to get started in Indie development
+ - Project management as single developer
+ - Testing and Continuous Integration
+ - How does an Indie developer decides which libraries or technologies to use?
+ - Designing without being a designer
+ - Play Store, in app purchases and subscriptions
+
+
+#### Host:
```
Comment (artem-zinnatullin) [inline show_notes/Episode_4.md]: Hosts

Comment (artem-zinnatullin) [inline show_notes/Episode_4.md]: Let's put guests first?

**Comment by artem-zinnatullin** [comment]
Few little things, otherwise LGTM

**Comment by sockeqwe** [comment]
changes applied

**Comment by artem-zinnatullin** [comment]
:+1: will try to publish episode in a few hours!

**Comment by sockeqwe** [comment]
👍

**Pull request merged**